.travis.yml

```diff
@@ -1,11 +1,11 @@
 language: node_js
 node_js:
   - 6
+  - 7
   - 8
   - 9
+os:
+  - linux
```

Comment:
Linux is already the default value for OS, there is no need to explicitly specify this.